src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import duke.storage.Storage;
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke extends DukeMethods {
```

Comment:
Essentially if the methods in DukeMethods are all static, you do not need duke to extend dukemethods